draft-ietf-jsonpath-base.md

```diff
@@ -449,13 +450,16 @@ return an error.
 
 ## Syntax
 
-Syntactically, a JSONPath consists of a root selector (`$`), which
-selects the root node of a JSON value, followed by a possibly empty
-sequence of *selectors*.
+Syntactically, a JSONPath consists of either a root selector (`$`), which
+selects the root node of a JSON value, or a current item selector (`@`), which
+selects a child node of a JSON value, followed by a possibly empty sequence of
+*selectors*.
```

Comment:
I think this could align with the syntax descriptor below better.

> Syntactically, a JSON Path consists of a start selector, which may either be the root selector `$` or the current item selector `@`, followed by...